src/test/java/Business/RoomTest.java

```diff
@@ -0,0 +1,76 @@
+package Business;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class RoomTest {
+
+    // Helper: because lastUpdated is formatted to seconds, we must wait > 1s
+    private void waitOneSecond() {
+        try { Thread.sleep(1100); } catch (InterruptedException ignored) {}
+    }
```

Comment:
Use a mock time service instead of Thread.sleep() to avoid slow tests.
```suggestion
    private void waitOneSecond() {
        try { Thread.sleep(1100); } catch (InterruptedException ignored) {}  //Mock the current time using a custom time service to avoid using Thread.sleep()
    }
```